### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by rofe** [adobe/helix-pipeline#522]
For screen readers to be able to efficiently navigate the HTML output of the pipeline - specifically `VDOMTransformer.getDocument()` - it should be accessible with landmarks, title and ARIA attributes by default. (For landmarks, we might want to consider using HTML5 tags like `main`, `nav`, etc. which would eliminate the need to introduce `role` attributes.)

References:
- https://www.w3.org/WAI/fundamentals/accessibility-principles/
- https://a11y-101.com/

**Comment by trieloff** [comment]
I feel like `div` vs. `main` or `nav` or `section` is reopening an old discussion, but I agree that we should have the correct accessibility markers by default.

**Comment by rofe** [comment]
Do you have a reference to the "old discussion"? I may not be the only one wondering why we aren't leveraging HTML5 out of the box.

**Comment by trieloff** [comment]
#279 is probably the best pointer.

**Comment by ramboz** [comment]
For what it's worth, my opinion on this is that Helix Pages should probably offer WCAG 2.0 AA level out of the box, same for the default HTL in the Helix demo site.

On the other hand, I'd expect devs using Helix directly (i.e. HTL & pre.js) to take care of all accessibility/SEO/microdata on their own mostly. Helix should not impose any semantics on the devs and let them have full control of the markup.

**Comment by tripodsan** [comment]
@rofe is there still something we should improve / change on the default HTML output of the pipeline ?

**Comment by rofe** [comment]
After the latest accessibility review of a helix proejct, I think we're good for now.
